app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant/AddFireHydrantDiameter.kt

```diff
@@ -0,0 +1,25 @@
+package de.westnordost.streetcomplete.quests.fire_hydrant
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement
+
+class AddFireHydrantDiameter : OsmFilterQuestType<Int>() {
+
+    override val elementFilter = "nodes with emergency = fire_hydrant and !fire_hydrant:diameter"
+    override val commitMessage = "Add fire hydrant diameter"
+    override val wikiLink = "Tag:emergency=fire_hydrant"
+    override val icon = R.drawable.ic_quest_fire_hydrant
+    override val isDeleteElementEnabled = true
+
+    override val questTypeAchievements = emptyList<QuestTypeAchievement>()
+
```

Comment:
The quest type should only be enabled in countries in which a custom layout which shows how the sign looks like as you did for Germany exists.

Reply:
See https://github.com/streetcomplete/StreetComplete/blob/48d694360b5b87157c1280cbc64aec1df7721458/app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cash/AddAcceptsCash.kt#L58 (obviously country codes will be likely different)